docs/maatregelen/2-owp-21-uitvoeren-audit-voor-naleving-vereisten.md

```diff
@@ -1,6 +1,7 @@
 ---
 # vul hier een titel in voor deze maatregel
 title: Neem het kunnen uitvoeren van een audit over de vereiste op in contractvoorwaarden en de contractovereenkomst 
```

Comment:
Kan wellicht korter?
```suggestion
title: Neem het kunnen uitvoeren van een audit over de vereiste op in de contractvoorwaarden 
```

Reply:
Het zijn twee verschillende zaken. Via contractvoorwaarden is vooraf kenbaar waaraan voldaan moet worden. Maar dit moet je vervolgens 'regelen' in de contractovereenkomst.